src/test/java/nextstep/lms/domain/FreeSessionTest.java

```diff
@@ -0,0 +1,60 @@
+package nextstep.lms.domain;
+
+import nextstep.users.domain.NsUserTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class FreeSessionTest {
+    FreeSession recruitingFreeSession;
+    FreeSession completedFreeSession;
+
+    @BeforeEach
+    void setting() {
+        recruitingFreeSession = new FreeSession(
+                "무료강의(모집중)",
+                LocalDateTime.of(2023, 10, 30, 0, 0),
+                LocalDateTime.of(2023, 12, 14, 23, 59),
+                new CoverImage("next.png", 0.8, 300, 200),
+                SessionStatus.RECRUITING,
+                new ArrayList<>());
+        completedFreeSession = new FreeSession(
+                "무료강의(모집마감)",
+                LocalDateTime.of(2023, 10, 30, 0, 0),
+                LocalDateTime.of(2023, 12, 14, 23, 59),
+                new CoverImage("next.png", 0.8, 300, 200),
+                SessionStatus.COMPLETED,
+                new ArrayList<>());
```

Comment:
테스트에서의 중복도 메서드를 활용해 분리해 보시면 어떨까요?
```suggestion
        recruitingFreeSession = createFreeSession("무료강의(모집중)", SessionStatus.RECRUITING);
        completedFreeSession = createFreeSession("무료강의(모집마감)", SessionStatus.COMPLETED);
```